### PR TITLE
fix directories kubeflow-pipelines frontend package

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

we noticed that we were using the wrong directories for the image. At the times we were working on that package we were also have lots of other packages to do, so, we missed that one and made mistakes about the directories, so, the directories should be and we should also include .dist folder:

- [kubeflow/pipelines@570e56d/frontend/Dockerfile#L30-L31](https://github.com/kubeflow/pipelines/blob/570e56dd09af32e173cf041eed7497e4533ec186/frontend/Dockerfile#L30-L31)
- 
- [kubeflow/pipelines@570e56d/frontend/Dockerfile#L39](https://github.com/kubeflow/pipelines/blob/570e56dd09af32e173cf041eed7497e4533ec186/frontend/Dockerfile#L39)

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: kubeflow-pipelines.yaml [hunk 1]
```diff
@@ -12,23 +12,23 @@ package:
...
       - kubectl
       - nodejs
-      - jq
-      - eslint
+      - py3-gpep517
+      - py3-pip
+      - py3-setuptools
+      - py3-wheel
+      - python3-dev
+      - wolfi-base
```
Comment (imjasonh) [inline kubeflow-pipelines.yaml:31]: Does this need to include apk? 🤔

**Review by imjasonh — APPROVED** [review]
Can you include a description of the issue being fixed (or link to issue), so it's clearer what the change is trying to accomplish?

**Comment by developer-guy** [comment]
> Can you include a description of the issue being fixed (or link to issue), so it's clearer what the change is trying to accomplish?

we noticed that we were using the wrong directories for the image. At the times we were working on that package we were also have lots of other packages to do, so, we missed that one and made mistakes about the directories, so, the directories should be and we should also include .dist folder:
> https://github.com/kubeflow/pipelines/blob/570e56dd09af32e173cf041eed7497e4533ec186/frontend/Dockerfile#L30-L31

> https://github.com/kubeflow/pipelines/blob/570e56dd09af32e173cf041eed7497e4533ec186/frontend/Dockerfile#L39

**Review by imjasonh — DISMISSED** [review]

File: kubeflow-pipelines.yaml (hunk 1 above)
Comment (cpanato) [reply kubeflow-pipelines.yaml:31]: @developer-guy need to check this comment here, if that is not needed we can resolve this for now

Comment (developer-guy) [reply kubeflow-pipelines.yaml:31]: I don't think that we need this one, let me remove it.

**Review by imjasonh — APPROVED** [review]

**Pull request merged**